openstack/cerebro/cerebro-main/values.yaml

```diff
@@ -0,0 +1,20 @@
+image:
+  pullPolicy: IfNotPresent
+  tag: SET-AS-HELM-PARAMETER # This should be set via --set to a git commit hash
+  registry: SET-AS-HELM-PARAMETER

```

Comment:
Please link the production ready image here already. Unless we need to deploy different versions in different regions, this is the place to control it.

Reply:
The actual image (`/cerebro/dc-state-service/`) is defined in each microservice  (see for example the [dc-state-service](https://github.com/ofrendo/helm-charts/commit/e1291b8d23c52fda9664e6f8e8e8de5407e21d86#diff-2c1ab44d7cdb73e54b3fa545079ac486574004447ab802002d4ce13337e9d11fR43). The `values.yaml` only defines the tag. 

- We can of course use `latest` for tag as the default. Would you prefer using `latest` for `tag` by default?`
- The registry should be Keppel. Should we pass the (internal) Keppel URL as default for `registry`?